dev/tasks/bundle/buildclassiceditor.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * Bundle configuration with hard coded set of features.
+ *
+ * At this moment we don't know a list of every dependency needed in the bundle. It is because
+ * editor features load automatically during initialization process. To work around this problem
+ * we have created a custom entry file where we defined some of imports with features
+ * needed to initialize editor.
+ */
+
+/**
+ * Babel helper.
+ *
+ * @TODO: Move to bundle task.
+ * Should be injected by bundle task, because our source code don't have to know that will be transformed.
+ * But it is not so easy. It is only possible to pass string with file path to rollup,
+ * so to do it we need to create a temporary file.
+ */
+import '../../../node_modules/regenerator-runtime/runtime.js';
```

Comment:
Would be cool if that could be resolved by some other way. This does not work if CKEditor is used as dependency (all ckeditor5 repos are in one node_modules folder). This is not a big issue I think but something to think about since bundling is not crucial for including CKEditor5 as dependency (in contrast to transpiling for tests).